Remove Samsung Pay Module (#874)
braintree/braintree_android: v5_MIGRATION_GUIDE.md

+       }
    }
}
```

## Samsung Pay

> The Samsung Pay integration has been removed. Please remove it from your app.

Reviewer: ```suggestion
The Samsung Pay integration is no longer supported. Please remove it from your app.
```
